src/main/java/com/github/manolo8/darkbot/core/entities/Hangar.java

```diff
@@ -0,0 +1,23 @@
+package com.github.manolo8.darkbot.core.entities;
+
+import java.util.ArrayList;
+
+public class Hangar {
+
+    private String hangarID;
+    private boolean hangar_is_active;
```

Comment:
I really dislike mixing-in sneak_case in java, i know this is because it's what the API returns, but i'd like to find a better way. Probably a GSON serializer can fix it

Reply:
Changed